client/components/DisplayTest/index.jsx

```diff
@@ -20,7 +20,17 @@ class DisplayTest extends Component {
     constructor(props) {
         super(props);
 
+        let {
+            test,
+            userId
+        } = this.props;
+
+        let conflicts = test.results
```

Comment:
We need to calculate this for each new test, so I'm pretty sure you can't put this here

Reply:
DisplayTest should be re-rendering every time it displays a new test.